src/tests/testData.js

```diff
@@ -0,0 +1,137 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const merge = require('lodash.merge');
+
+const carousel1 = {
```

Comment:
is it needed as there is a carousel1 in the test data below?

Reply:
It is defined at the top of the file so I can utilize data from that component inside of the final testData object. Otherwise I would not be able to reuse the common definition of `carousel1` component and would need to duplicate its data over all related to it definitions. Please check places where I do `carousel1xxxxx: merge({}, carousel1, { ... })`

Reply:
ok, thx, makes sense